scripts/std-lib-regression.sh

```diff
@@ -61,6 +61,7 @@ kani = {path=\"${KANI_DIR}/library/kani\"}
 cp ${KANI_DIR}/rust-toolchain.toml .
 
 echo "Starting cargo build with Kani"
+export RUST_BACKTRACE=1
```

Comment:
Is this intentional?

Reply:
Yes. I think we should always include that in our scripts to make sure we get a stack trace back when there's a crash.

I believe we do that in the regression script already, but it wouldn't cover when running the script on its own.